app/src/main/java/com/framgia/fsalon/screen/stylistbooking/StylistBookingFragment.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.fsalon.screen.stylistbooking;
+
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.framgia.fsalon.FSalonApplication;
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.data.source.ManageBookingRepository;
+import com.framgia.fsalon.data.source.UserDataSource;
+import com.framgia.fsalon.data.source.api.FSalonServiceClient;
+import com.framgia.fsalon.data.source.local.UserLocalDataSource;
+import com.framgia.fsalon.data.source.local.sharepref.SharePreferenceImp;
+import com.framgia.fsalon.data.source.remote.ManageBookingRemoteDataSource;
+import com.framgia.fsalon.data.source.remote.UserRemoteDataSource;
+import com.framgia.fsalon.databinding.FragmentStylistBookingBinding;
+
+/**
+ * StylistBooking Screen.
+ */
+public class StylistBookingFragment extends Fragment {
+    private StylistBookingContract.ViewModel mViewModel;
+
+    public static StylistBookingFragment newInstance() {
+        return new StylistBookingFragment();
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mViewModel = new StylistBookingViewModel(getActivity().getFragmentManager());
+        UserDataSource.RemoteDataSource remoteDataSource =
+            new UserRemoteDataSource(FSalonServiceClient.getInstance());
+        UserDataSource.LocalDataSource localDataSource =
+            new UserLocalDataSource(new SharePreferenceImp(FSalonApplication.getInstant()));
```

Comment:
ERROR: [STYLE][DLS_DEAD_LOCAL_STORE] Dead store to localDataSource in com.framgia.fsalon.screen.stylistbooking.StylistBookingFragment.onCreate(Bundle)
http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE